script.json.txt

```diff
@@ -0,0 +1,20 @@
+{
+name: Emilia Carvajal
+NickName: Emi
+age:  25
+address: Eudoro Galindo 1915
+department: 35C
```

Comment:
this line is not good for me, please change

Reply:
to me it is ok